pkg/kubectl/resource_printer.go

```diff
@@ -631,6 +634,21 @@ func printResourceQuotaList(list *api.ResourceQuotaList, w io.Writer) error {
 	return nil
 }
 
+func printServerStatus(item *api.ServerStatus, w io.Writer) error {
+	_, err := fmt.Fprintf(w, "%s\t%s\t%d\t%s\t%s\n", item.Component, item.Health, item.HealthCode, item.Msg, item.Err)
+	return err
+}
+
+func printServerStatusList(list *api.ServerStatusList, w io.Writer) error {
+	for _, item := range list.Items {
+		if err := printServerStatus(&item, w); err != nil {
```

Comment:
This was the pattern for the other resource printers, which I followed. Also, an error in the "printServerStatus" call would generally mean that there was an error in Fprintf, which means the data returned didn't quite match the expected types. Continuing to attempt to print the other lines seems...redundant (the given line probably wasn't printed as well).

A final argument against doing this is that the user may be able to get at the data by requesting an output of YAML or JSON.